src/Acuminator/Acuminator.Analyzers/StaticAnalysis/RowChangesInEventHandlers/RowChangesInEventHandlersAnalyzer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Acuminator.Analyzers.StaticAnalysis.EventHandlers;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.Roslyn.Syntax;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.FindSymbols;
+using PX.SM;
+
+namespace Acuminator.Analyzers.StaticAnalysis.RowChangesInEventHandlers
+{
+	public partial class RowChangesInEventHandlersAnalyzer : IEventHandlerAnalyzer
+	{
+		private static readonly ISet<EventType> AnalyzedEventTypes = new HashSet<EventType>()
```

Comment:
@dropsonic maybe we should add the supported event types to the interface similar to the SupportedDiagnostics and provide common check in the main analyzer?

Reply:
Nope because some analyzers contain a lot more complex logic which behaves differently for different event types.
And I don't see any performance benefits there.

Reply:
ok